ambuild2/run.py

```diff
@@ -143,6 +143,8 @@ def PreparerForAPI(api):
         from ambuild2.frontend.v2_0.prep import Preparer
     elif api == '2.1' or api.startswith('2.1.'):
         from ambuild2.frontend.v2_1 import Preparer
+    elif api == '2.2' or api.startswith('2.2.'):
+        from ambuild2.frontend.v2_2.prep import Preparer
     else:
         raise Exception('API version {0} not found'.format(api))
```

Comment:
Again, very painful to look at.
```suggestion
        from ambuild2.frontend.v2_0.prep import Preparer
    elif api == '2.1' or api.startswith('2.1.'):
        from ambuild2.frontend.v2_1 import Preparer
    elif api == '2.2' or api.startswith('2.2.'):
        from ambuild2.frontend.v2_2.prep import Preparer
    else:
        raise ImportError('API version {0} not found'.format(api))
```
Maybe a custom exception should be here as well.